handlers/event.go

```diff
@@ -134,8 +136,8 @@ func (h *Server) CreateEvent(w http.ResponseWriter, r *http.Request) error {
 // UpdateEvent updates an event by its id.
 func (h *Server) UpdateEvent(w http.ResponseWriter, r *http.Request) error {
 	w.Header().Add("Content-Type", "application/json")
-
-	event, err := h.eventById(r.PathValue("id"))
+	params := mux.Vars(r)
```

Comment:
Let's create a variable binding `id := params["id"]` to be consistent like in the other handlers.